Scripts/create_smet_from_netcdf/create_forcing.sh

```diff
@@ -27,8 +27,8 @@ echo "	Year ${yr}"
 # Module loading and basic setup
 mkdir -p log
 export LD_LIBRARY_PATH=$(pwd)/usr/lib/:${LD_LIBRARY_PATH}
-module purge
-ml intel; ml proj; ml netcdf
+# module purge
+# ml intel; ml proj; ml netcdf
```

Comment:
For my purposes (fully realize it can be different for other users), I would prefer if these lines were not commented.